neuralhydrology/evaluation/metrics.py

```diff
@@ -473,7 +473,7 @@ def fdc_flv(obs: DataArray, sim: DataArray, l: float = 0.3) -> float:
     r"""Calculate the low flow bias of the flow duration curve [#]_
     
     .. math:: 
-        \%\text{BiasFMS} = -1 \frac{\sum_{l=1}^{L}[\log(Q_{s,l}) - \log(Q_{s,L})] - \sum_{l=1}^{L}[\log(Q_{o,l})
+        \%\text{BiasFLV} = -1 * \frac{\sum_{l=1}^{L}[\log(Q_{s,l}) - \log(Q_{s,L})] - \sum_{l=1}^{L}[\log(Q_{o,l})
```

Comment:
nit-pick: could you change the `*` to `\times` for consistency?
```suggestion
        \%\text{BiasFLV} = -1 \times \frac{\sum_{l=1}^{L}[\log(Q_{s,l}) - \log(Q_{s,L})] - \sum_{l=1}^{L}[\log(Q_{o,l})
```

Reply:
Please take a look at the changes and see if they are good. (This is my first pull request on github. I am still new to it.